guide/english/blockchain/index.md

```diff
@@ -65,13 +65,13 @@ A third party is usually needed to validate trust relationships. In the most com
 For this reason, Blockchain is also known as **"The Trust Protocol"**.
 
 ## History
-The first work on a cryptographically secured chain of blocks was described in 1991 by Stuart Haber and W. Scott Stornetta. They wanted to implement a system where documents' timestamps could not be tampered with or backdated. In 1992, Bayer, Haber, and Stornetta incorporated Merkle trees to the design, which improved its efficiency by allowing several documents to be collated into one block.
+The first work on a cryptographically secured chain of blocks as described in 1991 by Stuart Haber and W. Scott Stornetta. They wanted to implement a system where documents' timestamps could not be tampered with or backdated. In 1992, Bayer, Haber, and Stornetta incorporated Merkle trees to the design, which improved its efficiency by allowing several documents to be collated into one block.
```

Comment:
> The first work on a cryptographically secured chain of blocks **was** described in 1991 by Stuart Haber and W. Scott Stornetta.

In this sentence, "was" acts as the verb. "as" can't work as a verb here. Please revert to "was".

You can add a hyphen here if you wish - "cryptographically secured" .